src/Data/Vec/Properties.agda

```diff
@@ -460,7 +460,7 @@ module _ {f : A → A → A} where
   zipWith-assoc assoc (x ∷ xs) (y ∷ ys) (z ∷ zs) =
     cong₂ _∷_ (assoc x y z) (zipWith-assoc assoc xs ys zs)
 
-  zipWith-idem : ∀ {f : A → A → A} → Idempotent _≡_ f →
+  zipWith-idem : Idempotent _≡_ f →
                  Idempotent {A = Vec A n} _≡_ (zipWith f)
```

Comment:
Would this be simpler as `Idempotent _≡_ (zipWith {n = n} f)`?

Reply:
Yes :+1: